src/bp/index.ts

```diff
@@ -76,6 +77,8 @@ try {
   require('dotenv').config({ path: path.resolve(process.PROJECT_LOCATION, '.env') })
   process.core_env = process.env as BotpressEnvironmentVariables
 
+  process.env.REDIS_CHANNEL_PREFIX = process.env.REDIS_CHANNEL_PREFIX || nanoid()
```

Comment:
That's not correct. If you forget to set REDIS_CHANNEL_PREFIX, you will get weird behavior if you have multiple nodes, with no easy way to detect that. I suggest we use "prod" by default, and on server startup, that we show the prefix and how many servers are "listening" to it

Reply:
Right! I added 'production' as the default value. I already print the prefix on boot so I do not think it is necessary to print the number of nodes listening on the same prefixed channels too. And I added the prefix to the diagnostic report.